lib/plugins/acts_as_customizable/lib/acts_as_customizable.rb

```diff
@@ -48,7 +48,7 @@ def self.included(base)
         end
 
         def available_custom_fields
-          CustomField.where("type = '#{self.class.name}CustomField'").sorted.to_a
```

Comment:
How does this behave after a new CustomField was created?

Reply:
this could have been cached only per request. Any new request would have already a new record.